src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
```

Comment:
The variable `employees` is not used in the file, so it can be removed. Unused variables can cause confusion for others reading your code and can lead to bugs. Always remove unused code to keep your codebase clean.